limitador/src/storage/redis/redis_async.rs

```diff
@@ -215,6 +215,14 @@ impl AsyncRedisStorage {
             .clone()
             .incr::<&str, i32, u64>("LIMITADOR_LIVE_CHECK", 1)
             .await
+            .map_err(|err| {
```

Comment:
This `is_alive(&self)` isn't used anymore afaict... but there is the `.then` wrt liveness_latency... So... what shall we do?

Reply:
I haven't found a ` #[allow(dead_code)] ` that might affect this (not complaining), also, method `is_alive` is not found in any other part of the project than here (method definition)

Reply:
@adam-cattermole what you recon we do? 🔥 the metric? Call it from a dedicated `Future`?

Reply:
I reckon we just burn the metric/method. We can add this back if needed but for now we could just use the `datastore_latency` from the batcher calls to redis (I know even this metric isn't perfect due to size of batch etc). If we decide we want this down the line it shouldn't be too hard to add it back